tox.ini

```diff
@@ -40,10 +40,9 @@ commands =
 changedir = {toxinidir}
 commands =
     # Use different configs for new (tuf/api/*) and legacy code
-    # TODO: configure black and isort args in pyproject.toml (see #1161)
-    black --check --diff --line-length 80 tuf/api tuf/ngclient
-    isort --check --diff --line-length 80 --profile black -p tuf tuf/api tuf/ngclient
-    pylint -j 0 tuf/api tuf/ngclient --rcfile=tuf/api/pylintrc
+    black tuf/api tuf/ngclient
+    isort --check --diff tuf/api tuf/ngclient
+    pylint -j 0 tuf/api tuf/ngclient --rcfile=pyproject.toml
```

Comment:
Let's also format and lint the scripts in the newly added `examples` directory. (see #1685)

Reply:
Will add a separate pr for that as I noticed there are a couple of mypy warnings that must be addressed as well.

Reply:
Oh right, we already have a ticket for that in #1697

Reply:
After we agree that this pr is good, I will submit a pr for #1685.